docs/source/app_developers_guide/creating_sawtooth_network.rst

```diff
@@ -258,16 +259,68 @@ a socket" in the `zmq-tcp API Reference <http://api.zeromq.org/4-2:zmq-tcp>`_.
 Step 1: Configure the Network on the First Node
 -----------------------------------------------
 
-.. important::
-
-   This step requires an existing application development environment as
-   described in :doc:`ubuntu`.
+   This step assumes an existing application development environment as
+   described in :doc:`ubuntu`. However, you will delete the existing blockchain
+   data, including the genesis block.
 
 #. If the first validator node is running, stop the validator, as described
    in the first step of :ref:`stop-sawtooth-ubuntu-label`.
 
-   If the first validator node is not running, start all Sawtooth components
-   except for the validator (see :doc:`docker`).
+#. If necessary, regenerate a validator private/public key pair in
```

Comment:
keypairs needn't be regenerated at all, if they have had an existing sawtooth network in dev mode or they've followed steps in app developer setup. This needs to be clarified.